### PR TITLE
added directus+supabase integration guide

### PR DESCRIPTION
**Pull request by erondpowell**
## What kind of change does this PR introduce?

Docs guide for integrating Directus + Supabase

## Additional context

I'm not 100% sure how `combined.json`, `gotrue.json`, and `storage.json` got changes. I'm just now seeing them.
I'm guessing when we run `make`?
Please let me know if I need to modify/gitignore these.

### CODE REVIEW

**Review by thorwebdev — CHANGES_REQUESTED** [review]
Thanks @erondpowell, yes, can you please revert the changes in `spec/*`? Thanks.

File: web/docs/guides/integrations/directus.mdx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+Supabase is an open-source Firebase alternative that provides a PostgreSQL database, storage, authentication, and a dynamic REST API based on your schema. While it is possible to self-host Supabase on your own infrastructure, this article will focus on Supabase Cloud’s Free tier, which is the fastest and easiest way to get started.
+
+![Directus App](/img/guides/integrations/directus/directus-20220608A.webp)
+
+Directus is an open-source data platform that layers on top of any SQL database, providing a powerful suite of tools. The Directus Engine provides dynamic REST and GraphQL APIs based on your schema, hooks and automation, authentication and access control, and file transformations. Directus Studio enables engineers and non-technical users alike to browse, manage, and visualize database content through a no-code app.
+
+---
+
+Supabase is strictly developer tooling, making it great for engineers, but its administration app is not suitable for non-technical business users. Directus, by comparison, is also developer tooling, but its Data Studio is safe and intuitive enough for anyone to use. This is the crucial bit that gives the two platforms such a strong “network effect.”
```
Comment (thorwebdev) [inline web/docs/guides/integrations/directus.mdx:21]:
```suggestion
Supabase is a suite of open-source tools making Postgres databases, file storage, authentication, edge functions and more accessible to developers of all skill levels. Directus is also developer tooling and additionally provides a Data Studio that is safe and intuitive enough for anyone, including non-technical users, to use. This is the crucial bit that gives the two platforms such a strong “network effect.”
```

**Review by thorwebdev — CHANGES_REQUESTED** [review]

**Comment by erondpowell** [comment]
> can you please revert the changes in `spec/*`? Thanks.

@thorwebdev  Done! This one's ready to roll out live, I think.

**Review by thorwebdev — APPROVED** [review]
LGTM, thanks.

@RamiroND @erondpowell you can merge this in whenever we're ready to release. Once merged it will be automatically deployed and published 👍

**Pull request merged**